### ISSUE TITLE
Why mAP degradation for StreamMapNet?

### ISSUE THREAD

**Issue by xiaolul2** [alfredgu001324/MapUncertaintyPrediction#3]
Thanks for your interesting work! I'd like to know why the learning of uncertainty has such a bad influence on mAP evaluation based on the  StreamMapNet framework. Is there any strategy for improvement? Thanks for your response.

**Comment by alfredgu001324** [comment]
Thank you for your interest in our work! That is actually one of the unresolved problems in our work. One of the possibilities is that I did not do a proper hyperparameter sweep (which is actually indeed the case). Another reason I was thinking is this uncertainty learning results in less stable training behaviour (as seen in the suppl of the paper), it is hard to stabilize the gradients during training. This might have a greater influence on a temporal aggregation setting. 

For now I do not really have a strategy for improvement since the main focus of the work is just investigating the relative performance improvements. But this can definitely be an exciting future research direction! (like using unc itself as additional feedback correcting signal during training process)
